recipes/ignition-math/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
```

Comment:
```suggestion
from conans.errors import ConanInvalidConfiguration

required_conan_version = ">=1.29.1"


```

due to `tools.remove_files_by_mask()`